instrumentation-core/spring/spring-boot-autoconfigure/README.md

```diff
@@ -0,0 +1,428 @@
+# OpenTelemetry Spring Auto-Configuration
+
+Auto-configures OpenTelemetry instrumentation for [spring-web](../spring-web-3.1/), [spring-webmvc](../spring-webmvc-3.1/), and [spring-webflux](../spring-webflux-5.0/). Leverages Spring Aspect Oriented Programming, dependency injection, and bean post-processing to trace spring applications. To include use all features listed below use the [opentelemetry-spring-starter](../starters/spring-starter/README.md).
+
+## Quickstart
+
+### Add these dependencies to your project.
+
+Replace `OPENTELEMETRY_VERSION` with the latest stable [release](https://mvnrepository.com/artifact/io.opentelemetry).
+ - Minimum version: `0.8.0`
+
+For Maven add to your `pom.xml`:
+
+```xml
+<dependencies>
+  <!-- opentelemetry -->
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-spring-boot-autoconfigure</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-api</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+   <!-- simple span exporter -->
+   <!-- outputs spans to console -->
+   <!-- provides opentelemetry-sdk artifact -->
+   <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-logging</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+
+</dependencies>
+```
+
+For Gradle add to your dependencies:
+
+```groovy
+//opentelemetry spring auto-configuration
+implementation 'io.opentelemetry.instrumentation:opentelemetry-spring-boot-autoconfigure:OPENTELEMETRY_VERSION'
+//opentelemetry
+implementation 'io.opentelemetry:opentelemetry-api:OPENTELEMETRY_VERSION'
+//opentelemetry exporter
+implementation 'io.opentelemetry:opentelemetry-exporters-otlp:OPENTELEMETRY_VERSION'
+```
+
+### Features
+
+#### Dependencies
+
+The following dependencies are optional but are required to use the corresponding features.
+
+Replace `SPRING_VERSION` with the version of spring you're using.
+ - Minimum version: `3.1`
+
+Replace `SPRING_WEBFLUX_VERSION` with the version of spring-webflux you're using.
+ - Minimum version: `5.0`
+
+Replace `SLF4J_VERSION` with the version of slf4j you're using.
+
+For Maven add to your `pom.xml`:
+
+```xml
+<dependencies>
+  <!-- opentelemetry exporters-->
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-jaeger</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-zipkin</artifactId>
+    <version>OPENTELEMETRY_VERSION</version>
+  </dependency>
+  <dependency>
+    <groupId>io.opentelemetry</groupId>
+    <artifactId>opentelemetry-exporters-logging</artifactId>
```

Comment:
did you mean for this to be otlp? (looks like otlp below in gradle config)

also, let's put otlp first in the exporters list, since it's our default

```suggestion
    <artifactId>opentelemetry-exporters-otlp</artifactId>
```